src/screens/LoginPage/LoginPage.test.tsx

```diff
@@ -448,8 +446,6 @@ describe('Testing Login Page Screen', () => {
     userEvent.type(screen.getByPlaceholderText('Password'), password.password);
 
     expect(password.password.length).toBeLessThan(8);
-
```

Comment:
1. You have removed a test
2. You have not tested both valid and invalid variations of passwords.

Please be more professional in the submission of your code changes

Reply:
I removed it because I replaced the tags it was testing:
``` 
{isInputFocused &&
         signformState.signPassword.length < 8 && (
         <span data-testid="passwordCheck">
          {t('atleast_8_char_long')}
         </span>
     )} 
     {!isInputFocused &&
          signformState.signPassword.length > 0 &&
            signformState.signPassword.length < 8 && (
           <span data-testid="passwordCheck">
          {t('atleast_8_char_long')}
       </span>
  )}
```
The code I wrote validates the length and password match while the above code only validates the password length. 
 I also went through the test file and saw there was already a test for when password and confirm password is not the same on line 151 of LoginPage.text.tsx.